.github/workflows/release.yaml

```diff
@@ -0,0 +1,27 @@
+name: Release
+on:
+  release:
+    types: [published, created, edited]
```

Comment:
are releases (via the github UI) the intended method for triggering this job? I ask because often people wish to have releases created via `git tag 'v1' && git push origin v1`. I also discovered this action if it helps: https://github.com/actions/create-release

Reply:
Yeah, that's what we do for truss-cli https://github.com/instructure-bridge/truss-cli/blob/master/.github/workflows/main.yml#L8

Reply:
if that's how we want to do it, that's fine. this was just in the github docs. i'll change it.